### ISSUE TITLE
Temporal Module - mean offsets

### ISSUE THREAD

**Issue by seamie6** [boschresearch/GaussianFlowOcc#2]
For training you:
Pass the Gaussian features and the temporal token to an MLP to predict offsets for each dynamic Gaussian
You then add this to the old Gaussian mean
Then you render these new Gaussians (sem and depth) and compute loss against pseudo-labels

Is this correct? Additionally, how do you restrict these offsets, as they appear to be raw logits, which you are adding to the real world means? Is it better to do it this way and let the model learn rather than use a sigmoid and restrict it to a certain range? Also, it seems ```movement_regularizer``` is disabled also.

Thank you.

**Comment by seamie6** [comment]
In the transformer for refining Gaussians, it is common to bound the XYZ offsets, so it is interesting that this doesn't seem to apply for the temporal module. And to clarify, when rendering the temporal samples, you render both dynamic and static objects as you do during time T? If so, have you experimented with just rendering dynamic objects, as this may enable the model to learn flow better, although it may worsen understanding of static objects.

**Comment by SBoeder** [comment]
Actually, I saw the same behavior for refining the Gaussians in the "Gaussian Rectifier". I am not actually bounding the offsets there either, as this has achieved better performance overall.
I am always rendering all the Gaussians for all frames, both the current one and all temporal frames. However, only the dynamic objects (meaning those, that have been classified as a dynamic object by the current model) are moved. I have not experimented rendering only the dynamic objects - however I believe this would not make a difference for learning flow, as the gradients for the Temporal Module are anyways only flowing through the dynamic Gaussians. 
When I refer to a temporal horizon of 2, this means I am rendering into [T-2, T-1, T, T+1, T+2].
